src/librustc_resolve/diagnostics.rs

```diff
@@ -93,6 +93,23 @@ mod bar {
 ```
 "##,
 
+E0253: r##"
+Attempt was made to import an unimportable value. This can happen when
+trying to import a method from a trait. An example of this error:
+
+```
+mod foo {
+    pub trait MyTrait {
+        fn do_something();
+    }
+}
+use foo::MyTrait::do_something;
+```
+
+In general, it's not legal to directly import methods belonging to a
```

Comment:
why "in general"? is there an exception? this feels like the diagnostic is not sure why it's firing. Let's not ship slightly ambiguous language like this